test/unit/User.test.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env mocha */
+import React from 'react';
+import sinon from 'sinon';
+import { expect } from 'chai';
+// Import the component that is going to be tested
+import User from './../../src/server/ApiRoutes/User.js';
+
+let mockTokenResponse = {};
+const mockRes = { redirect: () => {}, };
+
+const renderMockReq = (mockTokenResponse) => {
+  return {
+    get: (n) => { return n; },
+    protocol: 'http',
+    originalUrl: '/hold/request/b11995345-i14211097',
+    tokenResponse: mockTokenResponse,
+  };
+};
+
+describe('If requireUser does not receive valid value from "req.tokenResponse"', () => {
+  let requireUser;
+
+  before(() => {
+    requireUser = sinon.spy(User, 'requireUser');
+  });
+
+  after(() => {
+    mockTokenResponse = {
```

Comment:
Why is this in the `after` case? I don't like doing this because it means that the following tests depend on the previous tests. So if you remove the first couple of tests, the last tests will fail. _I think_ it's because you're assigning the `mockTokenResponse` object after the tests run. Try to make them independent. I know a lot of info will be repeated but they'll make for better tests.